src/components/Capture/index.js

```diff
@@ -27,6 +30,30 @@ export default class Capture extends Component {
 
   componentDidMount () {
     events.on('onMessage', (message) => this.handleMessages(message))
+    this.checkWebcamSupport();
+  }
+
+  checkWebcamSupport () {
+    DetectRTC.load( _ => {
+      this.setState({
+        DetectRTCLoading: false,
+        hasWebcam: DetectRTC.hasWebcam
+      })
+    });
+  }
+
+  supportsWebcam (){
+    if (DetectRTC.isGetUserMediaSupported && this.state.DetectRTCLoading) return true;
```

Comment:
Could this look a little cleaner as something like

```
const supportUnknown = DetectRTC.isGetUserMediaSupported && this.state.DetectRTCLoading;

// err on the side of webcam being supported
return supportUnknown || this.state.hasWebcam
```

Just to make it clearer why you're returning true when `DetectRTCLoading`